ci-operator/config/openshift/assisted-service/openshift-assisted-service-ocm-2.3.yaml

```diff
@@ -170,6 +170,16 @@ tests:
     env:
       IP_STACK: v4
     workflow: baremetalds-assisted-operator-ztp
+- as: e2e-metal-assisted-operator-ztp-periodic
+  cron: '@daily'
+  steps:
+    cluster_profile: packet
+    dependencies:
+      ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE: release:candidate
+      OPENSHIFT_INSTALL_RELEASE_IMAGE: release:candidate
```

Comment:
I think should remove so that it will use the nightly build.

Reply:
But maybe we should have 4.8 nightlies?
You know how to do that?